lib/patch-series.ts

```diff
@@ -858,14 +857,10 @@ export class PatchSeries {
     }
 
     protected async publishBranch(tagName: string): Promise<void> {
-        if (!this.project.publishToRemote || this.options.dryRun) {
-            return;
-        }
-
         if (this.options.redo) {
             tagName = "+" + tagName;
         }
-        await git(["push", this.project.publishToRemote,
+        await git(["push", `${this.project.publishToRemote}`,
```

Comment:
Is this really intentional?

Reply:
I had switched to the template literal because the linters decided publishToRemote could be null.  Reinstating the guard (as per your request) made the linters happy and the change has been removed.